app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
+
+<p>You can confirm your account email through the link below:</p>
```

Comment:
We may prefer a form here with a single button field and a label for the message

Reply:
Agreed, but this isn't currently being used since User model isn't :confirmable as of yet